components/usage-api/usage/v1/usage.proto

```diff
@@ -49,5 +49,6 @@ message BilledSession {
     google.protobuf.Timestamp start_time = 9;
     google.protobuf.Timestamp end_time = 10;
 
-    int64 credits = 11;
+    int64 credits_deprecated = 11 [deprecated=true];   // insufficient precision to represent credits for workspace time
```

Comment:
A proto field rename is backwards compatible, because the names are only used for nice API to access them but under the hood all fields are accessed by their field number (ie 11).

Reply:
Is there a need to deprecate here? The API hasn't seen production use yet so can we just remove the old field?

Reply:
It's being queried and we can't guarantee the order or the components we're rolling out. Could probs suffer some downtime but best we get into the habit of not breaking backwards compatibility